nextflow/workspace_description.md

```diff
@@ -0,0 +1,54 @@
+# Overview
+
+This notebook shows how to get started with [Nextflow](https://www.nextflow.io) on Enterprise Terra.
+
+# How do I get started?
+
+## Step 0: Set up your github SSH key and _Duplicate_ this workspace
+
+- In the Enterprise Terra UI, click on your profile icon, then click on "My Profile". Copy the public SSH key listed there.
+  Then, [add that key](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) to your GitHub account.
+  [This page](https://terra-docs.api.verily.com/docs/how_to_guides/terra_ssh_key_guide/) has more detail.
+
+  This setup allows the `terra-axon-examples` example repo, which you can see listed as a Git repository for the workflow, to be automatically cloned when you create a Cloud Environment.
+  > **Note**: if you do not have a GitHub account, then you can clone the repo manually after you've created the Cloud Environment.
+
+- Then, *Duplicate* this workspace. You can do that via the 'three-dot' menu in the upper right of the workspace.
+
+
+
+## Step 1: Preview prior runs of the relevant notebooks.
+
+Previews of this notebook, including outputs, are provided in the workspace resource folder "Notebook snapshots". Click the links listed below to view notebook previews in order to gain a better understanding of the setup and analysis included in this demonstration workspace:
+
+1. [Workspace setup] (https://terra-preprod-ui-terra.api.verily.com/workspaces/getting-started-with-workflows-workspace/resources/97ce452a-dd3c-4045-97d3-bc70db429f83/notebook_snapshots/workspace_setup.html)
+2. [Example 1: Run a Nextflow workflow](https://terra-preprod-ui-terra.api.verily.com/workspaces/getting-started-with-workflows-workspace/resources/21ffa484-75b9-46ff-b70d-6ebeb2962eb2/notebook_snapshots/nextflow_examples_first_exercise_fully_executed.html)
```

Comment:
I agree with Amy that we shouldn't link directly to the resource in the original workspace. Since relative links aren't possible at the moment, I also agree that directing the user to navigate to the Resources panel would be better.

Reply:
noted -- I have refactored this section accordingly